Composer/packages/client/src/components/GetStarted/GetStartedTask.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { Fragment } from 'react';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+
+import { NextSteps } from './GetStartedNextSteps';
```

Comment:
something is imported from GetStartedNextSteps and GetStartedNextSteps is importing something from GetStartedTask, that's a circular dependency. You need to move the reused types to a common types file that both import.

Reply:
good catch, moved the type out to a separate file.